src/harness/certs/generate_fake_certs.sh

```diff
@@ -234,6 +234,72 @@ openssl ca -cert sas_ca.cert -keyfile private/sas_ca.key -in server.csr \
     -policy policy_anything -extensions sas_client_mode_req_sign   -config ../../../cert/openssl.cnf \
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
+#Certificate for test case WINNF.FT.S.SCS.11
+echo "\n\nGenerate blacklisted client certificate/key"
+openssl req -new -newkey rsa:2048 -nodes \
+    -reqexts cbsd_req -config ../../../cert/openssl.cnf \
+    -out blacklisted_client.csr -keyout blacklisted_client.key \
+    -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Blacklisted CBSD "
+openssl ca -cert cbsd_ca.cert -keyfile private/cbsd_ca.key -in blacklisted_client.csr \
```

Comment:
What should be done with the CRL once it's generated? How can other certs be added to the CRL file?

Do we need to reference this specific CRL file when creating the cert? I don't think it's sufficient to give only the domain name, but I'm not quite sure.

Reply:
The   CRL chain file 'ca.crl' will be served by the simple CRL server submitted as   part of PR#469. This CRL server has an option to revoke certificate and   update the CRL chain file. Hence any certificate present in the 'certs'   directory can be revoked at any time using the menu options provided by   simple CRL server.